infrastructure/main.tf

```
@@ -56,7 +56,7 @@ locals {
    source        = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
    env           = var.env
    product       = var.product
-   name          = "${var.product}-v14-flexible"
+   name          = key_vault.resource_group_name
```

I meant the key name? 
```suggestion
   resource_group_name = key_vault.resource_group_name
```